src/install.rdf

```diff
@@ -3,7 +3,7 @@
     <Description about="urn:mozilla:install-manifest">
         <!-- General -->
         <em:id>swarth@franklindm</em:id>
-        <em:version>dev-build</em:version>
```

Comment:
Please leave the version as-is, this is automatically changed during release.

Reply:
Sorry. I don't know what to do. As you can see I uploaded via github web interface. I'm very bad with git. I don't know how to do what you requested.

Reply:
What does left the version as-is mean? Does it mean I should just remove this line?